app/models/manageiq/providers/lenovo/manager_mixin.rb

```diff
@@ -20,10 +20,11 @@ def connect(options = {})
     host       = options[:host] || address
     port       = options[:port] || self.port
     auth_type  = AUTH_TYPES[options[:auth_type]]
+    user_agent_label = "ManageIQ/#{MiqServer.first.version}"
```

Comment:
Once it always will be the same value and it isn't defined through options, I recommend it should be a constant.

Reply:
Ok, I can accept the constant. But it shouldn't be the `first.version`, because when the build change MIQ create another entry to this table. So, it should be `last.version`.

Reply:
I disagree with you @AndreyMenezes. This value is coming from a file named `VERSION` which is in the ManageIQ root, the point is: how to know what ManageIQ is instantiating the client, it could be the first, the last or another between both of them.

It is coming from this stack:

`manageiq/app/models/miq_server.rb`
```
class MiqServer < ApplicationRecord
(...)
     self.version    = Vmdb::Appliance.VERSION
end
(...)
```

`manageiq/lib/vmdb/appliance.rb`
```
module Vmdb
  module Appliance
    def self.VERSION
      @EVM_VERSION ||= File.read(File.join(File.expand_path(Rails.root), "VERSION")).strip
    end
(...)
end
end
```
 `manageiq/lib/vmdb/appliance.rb`


@rodneyhbrown7 @CharlleDaniel  is there a way to know what server is instantiating the client and then perform a kind of `MiqServer.find` and then getting the `.version` variable?

Reply:
@walteraa I got it! the `find` should be perfect! I'm thinking we only can have the last build server, but I'm wrong!